app/modules/annotations/models.py

```diff
@@ -495,3 +495,43 @@ def get_elasticsearch_schema(cls):
         from app.modules.annotations.schemas import AnnotationElasticsearchSchema
 
         return AnnotationElasticsearchSchema
+
+    # this initiates ID pipeline *for this single* annotation.  it is roughly analogous to sighting.ia_pipeline()
```

Comment:
I'm really not sure this should be in the annot module. It seems to be doing the same as the ia_pipeline code in the sightings module on a subset of annots, in this case the subset is one but as this uses all the code in the sightings module to send the request and handle the response, the annot module feels like the wrong place to be initiating it.

Reply:
Update: Given the changes I'm currently doing for retries, this functionality now definitely needs to be in the sighting module and not in annotation module.